dvc/fs/dvc.py

```diff
@@ -320,7 +320,7 @@ def ls(  # pylint: disable=arguments-differ # noqa: C901
             with suppress(FileNotFoundError, NotADirectoryError):
                 for info in dvc_fs.ls(dvc_path, detail=True):
                     dvc_infos[dvc_fs.path.name(info["name"])] = info
-            dvc_exists = bool(dvc_infos) or dvc_fs.exists(dvc_path)
+                dvc_exists = True
```

Comment:
If we reach this line, it means the path exists (otherwise it will hit one of the suppressed errors above). @efiop Any reason we use `suppress` here but `try/except` below on lines 331-338?

Reply:
No particular reason, just was written during different times. Just lack of consistency.